client/src/templates/Challenges/components/hotkeys.css

```diff
@@ -1,4 +1,3 @@
 /* hide the outline for the HotKeys component */
-div[tabindex='-1']:focus {
```

Comment:
Sorry, perhaps I didn't do my suggested fix properly. This line should not be deleted. It still needs to be

```
div[tabindex='-1']:focus {
```

Reply:
There we go. I should have known better than to just hit "commit suggestion".